// src/datos.ts
// Definición de tipos de datos
export type Persona = {
    ID: number;
    Nombre: string;
    Identificación: string;
  };
  
  export type Encuesta = {
    ID: number;
    Descripción: string;
    Detalles: string;
  };
  
  export type Registro = {
    ID: number;
    IDPersona: number;
    IDEncuesta: number;
    Fecha: string;
    Hora: string;
    Ubicación: string;
  };
  
  // Arreglos de objetos
  export const personas: Persona[] = [
    { ID: 1, Nombre: "Juan", Identificación: "123456789" },
    { ID: 2, Nombre: "María", Identificación: "987654321" },
    { ID: 3, Nombre: "Pedro", Identificación: "456789123" },
    { ID: 4, Nombre: "Ana", Identificación: "789123456" },
    { ID: 5, Nombre: "Luis", Identificación: "321654987" }
  ];
  
  export const encuestas: Encuesta[] = [
    { ID: 1, Descripción: "Encuesta 1", Detalles: "Detalles de la encuesta 1" },
    { ID: 2, Descripción: "Encuesta 2", Detalles: "Detalles de la encuesta 2" },
    { ID: 3, Descripción: "Encuesta 3", Detalles: "Detalles de la encuesta 3" },
    { ID: 4, Descripción: "Encuesta 4", Detalles: "Detalles de la encuesta 4" },
    { ID: 5, Descripción: "Encuesta 5", Detalles: "Detalles de la encuesta 5" }
  ];
  
  export const registros: Registro[] = [
    { ID: 1, IDPersona: 1, IDEncuesta: 3, Fecha: "2024-04-25", Hora: "10:00", Ubicación: "Ubicación 1" },
    { ID: 2, IDPersona: 2, IDEncuesta: 1, Fecha: "2024-04-25", Hora: "11:30", Ubicación: "Ubicación 2" },
    { ID: 3, IDPersona: 3, IDEncuesta: 5, Fecha: "2024-04-25", Hora: "12:45", Ubicación: "Ubicación 3" },
    { ID: 4, IDPersona: 4, IDEncuesta: 2, Fecha: "2024-04-25", Hora: "14:15", Ubicación: "Ubicación 4" },
    { ID: 5, IDPersona: 5, IDEncuesta: 4, Fecha: "2024-04-25", Hora: "15:30", Ubicación: "Ubicación 5" }
  ];
